### PR TITLE
[Vanilla Stage Update] - Removed and updated front-facing mentions of DMS

### PR DESCRIPTION
**Pull request by MitchellShiell**
## Summary

- This removes and updates all front-facing mentions of the DMS

- Stage is not just a DMS UI it is the front end portal interface for all Overture platforms and as such the language should reflect its generalizability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,15 +45,15 @@ For more information about the DMS-UI, including setup and how to integrate Over
   npm run dev
 ```
 
-Once running the DMS-UI is available from your [http://localhost:3000](http://localhost:3000)
+Once running Stage is available from your [http://localhost:3000](http://localhost:3000)
```
Comment (justincorrigible) [inline README.md:48]:
```suggestion
Once running, Stage is available from your [http://localhost:3000](http://localhost:3000)
```

File: README.md [hunk 2]
```diff
@@ -76,10 +76,10 @@ Check out our [Next.js deployment documentation](https://nextjs.org/docs/deploym
 
 ## Support & Contributions
 
-- Filing an [issue](https://github.com/overture-stack/dms-ui/issues)
-- Making a [contribution](CONTRIBUTING.md)
-- Connect with us on [Slack](https://overture-bio.slack.com/)
-- Add or Upvote a [feature request](https://github.com/overture-stack/dms-ui/issues?q=is%3Aopen+is%3Aissue+label%3Anew-feature+sort%3Areactions-%2B1-desc)
+- Filing an [issue](https://github.com/overture-stack/Stage/issues)
```
Comment (justincorrigible) [inline README.md:79]:
```suggestion
- Filing an [issue](https://github.com/overture-stack/stage/issues)
```

File: README.md [hunk 3]
```diff
@@ -89,21 +89,19 @@ Check out our [Next.js deployment documentation](https://nextjs.org/docs/deploym
...
+| [Ego](https://github.com/overture-stack/ego)           | An authorization and user management service                                                      |
+| [Ego UI](https://github.com/overture-stack/ego-ui)     | A UI for managing Ego authentication and authorization services                                   |
+| [Score](https://github.com/overture-stack/score)       | Transfer data to and from any cloud-based storage system                                          |
+| [Song](https://github.com/overture-stack/song)         | Catalog and manage metadata associated to file data spread across cloud storage systems           |
+| [Maestro](https://github.com/overture-stack/maestro)   | Organizing your distributed data into a centralized Elasticsearch index                           |
+| [Arranger](https://github.com/overture-stack/arranger) | A search API with reusable UI components that build into configurable and functional data portals |
+| [Stage](https://github.com/overture-stack/Stage)       | A simple web browser UI that integrates Ego and Arranger                                          |
```
Comment (justincorrigible) [inline README.md:104]:
```suggestion
| [Stage](https://github.com/overture-stack/stage)       | A simple web browser UI that integrates Ego and Arranger                                          |
```

File: components/Head.tsx [hunk 4]
```diff
@@ -25,7 +25,7 @@ import NextHead from 'next/head';
 const PageHead = ({ subtitle }: { subtitle?: string }) => {
 	return (
 		<NextHead>
-			<title>Overture DMS{subtitle ? ` - ${subtitle}` : ''}</title>
+			<title>Overture {subtitle ? ` - ${subtitle}` : ''}</title>
```
Comment (justincorrigible) [inline components/Head.tsx:28]:
```suggestion
			<title>Overture Stage{subtitle ? ` - ${subtitle}` : ''}</title>
```

File: components/pages/explorer/getConfigError.tsx [hunk 5]
```diff
@@ -82,32 +82,15 @@ const getConfigError = ({
...
-			<span
-				css={css`
-					font-weight: bold;
-				`}
-			>
-				config.yaml
-			</span>{' '}
-			file during installation and have been used to create your Arranger server configs.{' '}
-			<GenericHelpMessage />
+			One or more of the following values required by the platform do not exist. Please make sure
```
Comment (justincorrigible) [inline components/pages/explorer/getConfigError.tsx:92]: platform or portal? shouldn't this be consistent with line 85?

File: components/Head.tsx (hunk 4 above)
Comment (MitchellShiell) [reply components/Head.tsx:28]: I placed the stage title in the lab name, the title is a bit more hardcoded so I left it generic (meaning saving us changes needed on demo/quickstart)

File: components/pages/explorer/getConfigError.tsx (hunk 5 above)
Comment (MitchellShiell) [reply components/pages/explorer/getConfigError.tsx:92]: updated to platform

File: components/Head.tsx (hunk 4 above)
Comment (justincorrigible) [reply components/Head.tsx:28]: Sounds good. let's remove that trailing space then...
```suggestion
			<title>Overture{subtitle ? ` - ${subtitle}` : ''}</title>
```

**Review by justincorrigible — APPROVED** [review]
Awesome!! Ready to "Squash and Merge". :shipit:

**Pull request merged**